src/app/pages/HomePage/index.test.tsx

```diff
@@ -21,22 +22,31 @@ const homePageData = {
   },
 };
 
+const afriqueHomePageData = {
+  title: afriqueHomePageDataFixture.title,
+  description: afriqueHomePageDataFixture.description,
+  curations: afriqueHomePageDataFixture.curations,
+  metadata: {
+    ...afriqueHomePageDataFixture.metadata,
+    type: 'home',
+  },
+};
```

Comment:
We should be able to do this, and then use `afriqueHomePageDataFixture` instead of `afriqueHomePageData` below

```suggestion
afriqueHomePageDataFixture.metadata.type = 'home'
};
```